contracts/payloads/IGP68/PayloadIGP68.sol

```diff
@@ -0,0 +1,587 @@
+
+            // TODO: set max supply shares limits?
+        }
+    }
+
+    /// @notice Action 2: Set dust allowance to rsETH-ETH<>wstETH, rsETH<>wstETH, weETHs-ETH<>wstETH vaults
+    function action2() internal {
+        {
+            // [TYPE 2] rsETH-ETH<>wstETH | Smart collateral & debt
+            Vault memory VAULT_rsETH_ETH_AND_wstETH = Vault({
+                vault: getVaultAddress(78),
+                vaultType: TYPE.TYPE_2,
+                supplyToken: address(0),
+                borrowToken: wstETH_ADDRESS,
+                baseWithdrawalLimitInUSD: 0, // $0
+                baseBorrowLimitInUSD: 7_500_000, // $7.5M
+                maxBorrowLimitInUSD: 18_000_000 // $18M
+            });
+
+            setVaultLimits(VAULT_rsETH_ETH_AND_wstETH); // TYPE_2 => 78
+
+            VAULT_FACTORY.setVaultAuth(
+                getVaultAddress(78),
+                TEAM_MULTISIG,
+                false
+            );
+        }
+
+        {
+            // [TYPE 1] rsETH<>wstETH | collateral & debt
+            Vault memory VAULT_rsETH_AND_wstETH = Vault({
+                vault: getVaultAddress(79),
+                vaultType: TYPE.TYPE_1,
+                supplyToken: rsETH_ADDRESS,
+                borrowToken: wstETH_ADDRESS,
+                baseWithdrawalLimitInUSD: 20_000_000, // $20M
+                baseBorrowLimitInUSD: 7_500_000, // $7.5M
+                maxBorrowLimitInUSD: 18_000_000 // $18M
+            });
+
+            setVaultLimits(VAULT_rsETH_AND_wstETH); // TYPE_1 => 79
+
+            VAULT_FACTORY.setVaultAuth(
+                getVaultAddress(79),
+                TEAM_MULTISIG,
+                false
+            );
+        }
+
+        {
+            // [TYPE 2] weETHs-ETH<>wstETH | Smart collateral & debt
+            Vault memory VAULT_weETHs_ETH_AND_wstETH = Vault({
+                vault: getVaultAddress(80),
+                vaultType: TYPE.TYPE_2,
+                supplyToken: address(0),
+                borrowToken: wstETH_ADDRESS,
+                baseWithdrawalLimitInUSD: 0, // $0
+                baseBorrowLimitInUSD: 7_500_000, // $7.5M
+                maxBorrowLimitInUSD: 18_000_000 // $18M
+            });
+
+            setVaultLimits(VAULT_weETHs_ETH_AND_wstETH); // TYPE_2 => 80
+
+            VAULT_FACTORY.setVaultAuth(
+                getVaultAddress(80),
+                TEAM_MULTISIG,
+                false
+            );
+        }
+    }
+
+    /// @notice Action 3: Increase ETH-USDC Dex Pool and Vaults Limits
+    function action3() internal {
+        address ETH_USDC_DEX_ADDRESS = getDexAddress(12);
+
+        {
+            // ETH-USDC Dex Limits
+            Dex memory DEX_ETH_USDC = Dex({
+                dex: ETH_USDC_DEX_ADDRESS,
+                tokenA: ETH_ADDRESS,
+                tokenB: USDC_ADDRESS,
+                smartCollateral: true,
+                smartDebt: true,
+                baseWithdrawalLimitInUSD: 30_000_000, // $30M
```

Comment:
this seems excessive to me. AFAIK the idea was to set this as max supply shares instead by setting 15M shares as you do below, but base withdrawal limit itself I would keep at like 20M?